content/docs/user-patterns/document-profile/index.md

```diff
@@ -0,0 +1,67 @@
+
+Publish a new version of the document using special attributes interpreted by RKVST for this event type.
+
+| Event Attributes         | Meaning                                         | Requirement                               |
+|--------------------------|-------------------------------------------------|-------------------------------------------|
+| arc_display_type         | tells RKVST how to interpret Event              | Required, must be set to `Publish`        |
+| document_version_authors | list of authors on this version of the document | Optional, if present must be list of maps |
+
+| Asset Attributes      | Meaning                                                                        | Requirement                                     |
+|-----------------------|--------------------------------------------------------------------------------|-------------------------------------------------|
+| document_document     | attachment containing this version of the document | Optional, can use hash only                     |
+| document_hash_value   | hash of the most this version of the document                                | Required                                        |
+| document_hash_alg     | algorithm used for hashing. We only officially support SHA-256.                | Required
+| document_version      | specific version string for the most recent version of the document            | Optional                                        |
+| document_status       | label for filtering and accommodating critical document lifecycle events       | Required, must be `Published` |
+
+### Withdraw Event
+
+Withdraw an entire document (mark that it is no longer considered current).
+
+| Event Attributes                | Meaning                                         | Requirement                               |
+|--------------------------|-------------------------------------------------|-------------------------------------------|
+| arc_display_type         | tells RKVST how to interpret Event                    | Required, must be set to `Withdraw`        |
+| document_withdraw_reason | reason why document has been withdrawn | Optional, should be one of `Retired` or `Published In Error` |
+
+| Asset Attributes             | Meaning                                                                        | Requirement                                                        |
+|------------------------------|--------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| document_document            | attachment that points to the most recent version of the document being traced | Optional, can use hash only                                        |
+| document_hash_value          | hash of the most recent version of the document                                | Optional, must be present if `document_latest_document` is present |
```

Comment:
```suggestion
| document_hash_value          | hash of the most recent version of the document                                | Optional, must be present if `document_document` is present |
```

Reply:
Actually this makes limited sense in general, as does attaching a withdrawn document